packages/collector/test/tracing/protocols/grpc-js/test.js

```diff
@@ -17,7 +16,7 @@ const globalAgent = require('../../../globalAgent');
 
 const agentControls = globalAgent.instance;
 
-const mochaSuiteFn = semver.satisfies(process.versions.node, '>=8.13.0') ? describe : describe.skip;
+const mochaSuiteFn = describe;
```

Comment:
_nitpick:_
Consider using `describe` directly instead of  `mochaSuiteFn`, as there's no need for an additional alias in this case.